docs/documentation.md

```diff
@@ -17,6 +17,124 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+# RabbitMQ Documentation
+
+Welcome to RabbitMQ documentation. Use the drop-down menu on the top right of any page to select the documentation version for the RabbitMQ release that you are using. We update each version of documentation with the latest minor and patch information for that release.
+
+For previous 3.12.x documentation, select the **3.12** option from the drop-down menu. 3.12.x documentation is hosted on our previous RabbitMQ site. This documentation is not versioned. 
```

Comment:
I don't think we need this paragraph as the previous one already covered the fact that other versions are available in the list.

Also, "This documentation is not versioned" is not true. It is versioned: the website contains the docs for 3.12.x (but only that one).